docs/scaling.md

```diff
@@ -0,0 +1,97 @@
+# Scaling your database deployment
+
+Scaling in Percona Everest allows for easy adjustment of database capacity and performance to meet the changing demands.
+
+## Vertical scaling
+            
+Vertical scaling is essential for addressing increased workloads or performance demands. You can scale up or scale down as per your requirements. 
+        
+* Scale up
+
+    You can vertically scale up your Percona Everest instance by modifying its configuration to a higher performance tier. This involves increasing the memory and/or CPU threshold. To improve the performance of your cluster, you can create a cluster with a bigger disk size and memory that can hold more data.        
```

Comment:
A bigger disk won't increase performance. Also, the user can't change this value when editing the the DB cluster so it makes no sense to mention this in the scaling page.

Reply:
Fixed.